### DISCUSSION TITLE
Required confirm Webauthn Or Otp when create new realm

### DISCUSSION THREAD

**Discussion by uuhnaut69** [dasniko/keycloak-extensions-demo#8] [Q&A] [answered]
Hi, I wanna implement that when admin user perform action delete or create new client/realm, they have to confirm by OTP or Webauthn. How can i implement it?

**Comment by dasniko** [comment, marked as answer]
This is about authentication, not authorization.

**Reply by uuhnaut69** [reply]
Thanks for your reply, can you give some hint to implement it ?
